python/dgl/graph.py

```diff
@@ -3401,9 +3401,13 @@ def local_var(self):
         DGLGraph
             The graph object that can be used as a local variable.
         """
+        local_node_frame = FrameRef(Frame(self._node_frame._frame))
```

Comment:
Better explain the behavior of initializer in the docstring.

Reply:
Better?

Reply:
I mean users should know the `local_var` use the same node/edge initializer as the original `DGLGraph`, maybe you could mention it in the Notes section.

Reply:
Done.